app/domain/logs.rb

```diff
@@ -154,8 +154,8 @@ module AuthnK8s
         code: "CONJ00015D"
       )
 
-      CopySSLToPodSuccess = ::Util::TrackableLogMessageClass.new(
-        msg:  "Copied SSL certificate successfully",
+      InitializeCopySSLToPodSuccess = ::Util::TrackableLogMessageClass.new(
+        msg:  "Initialized SSL certificate copy successfully",
```

Comment:
@shulifink please review. 
This message will be written after we started the cert copy. it doesn't indicate a successful copy as we don't wait for the cert to be injected.

Reply:
Started copying the SSL certificate successfully...

Reply:
or 

Successfully started copying the SSL certificate...

Reply:
I'm doubting if it's appropriate to use by `SSL` acronym there.
SSL relates to Socket Layer Security deprecated predecessor of TLS - Transport Layer Security.
And using TLS in those sentences does not make sense.
I think that we need to remove `SSL` at all.

Reply:
So `Started copying the certificate successfully...`?

Reply:
or `Started copying the client certificate successfully...`?

Reply:
One of them :-)